python/modules/map_plate.py

```diff
@@ -41,95 +76,51 @@ def _getting_paths_mp(input_path):
         if path not in subfile_list and path[-4:] == ".csv":
             param_paths.append(path)
     return param_paths
-    
-def _making_path_list(main_path, file_list): # maybe this should be in file_managment?? gonna change method of path joining to more safe (not simple strings merge)
-    path_list = []
-    for filepath in file_list:
-        path = main_path + filepath
-        path_list.append(path)
-    return path_list
 
-def _parsing_matrix(path, deltimer, exp_part):
-    active_wells = []
-    data = CSV_M.read_csv(path, deltimer)
-    for row in range(len(data)):
-        for col in range(len(data[row])):
-            if data[row][col] == exp_part:
-                nums = [row, col]
-                active_wells.append(nums)
-    return active_wells
+def _collect_exp_settings(abs_path, mp_file, delimiter, mp_dict):
+    rel_path = FM.path_get_relative(abs_path, mp_file)
+    fullname = ".".join(FM._path_split(rel_path))
+    extension_length = len(FM.file_get_extension(fullname))  #extracting proper column name from rel_path i.e. compare.1.1 from compare/1.1.csv
+    name = fullname[:-extension_length]
+    data = CSV_M.read_csv(mp_file, delimiter)
+    for well in mp_dict:
+        position = mp_dict[well]["position"]
+        value = (data[position[0]][position[1]])
+        mp_dict[well][name] = value
 
-def _parsing_map_plate(mp_path, paths_mp, deltimer, exp_part, exp_name, exp_parts_all):
-    param_dict = {}
-    active_path = FM.path_join(mp_path, "args_active.csv")
-    names_path = FM.path_join(mp_path, "args_ind.csv")
-    positions_path = FM.path_join(mp_path, "args_names.csv")
-    if FM.path_check_existence(positions_path):
-        tmp = True
-    active_wells = _parsing_matrix(active_path, deltimer, exp_part)
-    for well in active_wells:
-        name = _collecting_exp_settings(names_path, well, deltimer)
-        result = []
-        if tmp == True:
-            result.append(_collecting_exp_settings(positions_path, well, deltimer))
-        else:
-            result.append(name)
-        result.append(exp_name)
-        result.append(exp_parts_all)
-        result.append(exp_part)
-        for mp_file in paths_mp:
-            param = _collecting_exp_settings(mp_file, well, deltimer)
-            result.append(param)
-        param_dict[name] = deltimer.join(result)
-    return param_dict
-  
-def _collecting_exp_settings(mp_file, well, deltimer):
-    row_nr, col_nr = well
-    data = CSV_M.read_csv(mp_file, deltimer)
-    print(mp_file)
-    print(row_nr)
-    print(col_nr)
-    return (data[row_nr][col_nr])
-    
-def _getting_column_titles(abs_path, paths):
+def get_param_value(mp_dict, well_name, param): # get values for a given well and parameter
```

Comment:
Apply changes to functions comments format, according Google Python Style Guide https://google.github.io/styleguide/pyguide.html#Comments